Makefile

```
@@ -184,6 +184,10 @@ generate-deploy:
 	go mod vendor
 	$(MAKE) -C deploy
 
+.PHONY: check-deploy-committed
+check-deploy-committed: ## Fail in case there are uncommitted changes for deploy directory
+	test -z "$(shell git status --short)" || (echo "files were modified: " ; git status --short ; false)
```

```suggestion
.PHONY: check-all-committed
check-all-committed: ## Fail in case there are uncommitted changes
	test -z "$(shell git status --short)" || (echo "files were modified: " ; git status --short ; false)
```
This is checking for the whole project not for just the deploy folder, can you please update the comment to reflect the same?

---

IIUC, this command checks for yamls only is deploy directory and not the whole project. So, that's the reason use this name.

---

i dont see any specific check for .yaml files. `git status --short` check for uncommitted files in the repo.

---

I meant the previous command that is run before this check is done.
That command only makes the update in the deploy directory.

---

we need to adjust the CI/action to run this at last part to ensure there are no uncommitted changes or run this after generating the deploy but we should not tie this makefile target with deploy in name as its not only meant to check changes in deploy